docs/customization_guide/inference_protocols.md

```diff
@@ -145,24 +146,31 @@ protocol types mentioned above:
   * `trace` : [trace endpoint](https://github.com/triton-inference-server/server/blob/main/docs/protocol/extension_trace.md).
   * `logging` : [logging endpoint](https://github.com/triton-inference-server/server/blob/main/docs/protocol/extension_logging.md).
 
-* `restricted-key` : Key to determine the GRPC request header to be checked when a
-request to the protocol is received. The completed header will be in the form of
-`triton-grpc-protocol-<restricted-key>`
+* `restricted-key` : The GRPC / HTTP request header
+to be checked when a request is received. The
+completed header for GRPC will be in the form of
+`triton-grpc-protocol-<restricted-key>`. The completed header for HTTP
+will be in the form of `<restricted-key>`.
```

Comment:
Just a question, not expecting any changes. Any reason we didn't follow similar format for HTTP like `triton-http-api-<restricted-key>` ?

Reply:
Not really. In hindsight it might have been better to follow the same pattern. I think I was confused at why we prefixed a fixed string instead of just using the passed in header value. We can maybe revisit the behavior of grpc.